#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-recordings-refresh.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change include path for svdrpservice.h so it works with vdr-dev

@DPATCH@
Index: svdrp.h
===================================================================
--- vdr-plugin-remotetimers-0.1.1.orig/svdrp.h	(revision 549)
+++ vdr-plugin-remotetimers-0.1.1/svdrp.h	(working copy)
@@ -9,7 +9,7 @@
 #define _REMOTETIMERS_SVDRP__H
 
 #include <stdlib.h>
-#include "../svdrpservice/svdrpservice.h"
+#include "svdrpservice.h"
 #include <vdr/tools.h>
 #include <vdr/plugin.h>
 
